### DISCUSSION TITLE
Define a new named color?

### DISCUSSION THREAD

**Discussion by Ampa** [Textualize/rich#1200] [Q&A] [answered]
Is it possible to define a new named color, that can be used in the same way as the standard colors?

Example: I have a color called 'pixie' which can be applied as "[pixie]Hello World[/]".

Thanks.

**Comment by willmcgugan** [comment, marked as answer]
Yes. See the docs on [themes](https://rich.readthedocs.io/en/latest/style.html#style-themes).

**Reply by Ampa** [reply]
Thank you for the pointer. Got it figured out.
